content/docs/next-release/deployment.md

```diff
@@ -398,6 +398,7 @@ Jaeger supports gRPC based storage plugins. For more information refer to [jaege
 
 Available plugins:
 
+* [Logz.io](https://github.com/logzio/jaeger-logzio)
```

Comment:
- please move into second place
- this is not an open-source backend, correct? Please clarify that this is commercially hosted storage

Reply:
@yurishkuro is this clear enough?